sys/include/net/sock/util.h

```diff
@@ -49,6 +50,7 @@ int sock_udp_ep_fmt(const sock_udp_ep_t *endpoint, char *addr_str, uint16_t *por
  * @note Caller has to make sure hostport and urlpath can hold the results!
  *       Make sure to provide space for SOCK_HOSTPORT_MAXLEN respectively
  *       SOCK_URLPATH_MAXLEN bytes.
+ *       Schema part of the URL is limited to SOCK_SCHEMA_MAXLEN.
```

Comment:
How is the reader supposed to know now what value `SOCK_SCHEMA_MAXLEN` is? I guess the most sensible solution would be to expose the variable here again below with the other macros (sorry for making you move it first).